README.md

```diff
@@ -228,7 +228,7 @@ APIs!**
 ### Operating a private network
 
 Maintaining your own private network is more involved as a lot of configurations taken for
-granted in the official networks need to be manually set up.
+granted the official networks need to be manually set up.
```

Comment:
What? Definitely better as it was before

Reply:
Ok.